common/arduino/variant.c

```diff
@@ -24,6 +24,8 @@
 #define PRINT           printk
 #endif
 
+#define PINMUX_NAME CONFIG_PINMUX_NAME
```

Comment:
That doesn't mean it's required, or good. My opinion is that we should remove it.  We're not gaining any abstraction or de-cluttering any of the code by using that define. It's just an extra, unnecessary layer of indirection that requires extra effort from the reader. I feel the same about "printk-->PRINT" but that's an unrelated matter. So I won't approve that, unless you have a good reason for it that you can share?